Providers/Scripts/2.6x-2.7x/Scripts/nxFile.py

```diff
@@ -1065,16 +1067,34 @@ def SetProxyFromConf():
             if 'http:' in info:
                 os.environ['HTTP_PROXY'] = info
     return
+
+def GetRemoteFileWithRetries(fc):
```

Comment:
>ie [](start = 25, length = 2)

Does any of the OMS solution uses remote location to download files? If not we are good.